base/dates/Dates.jl

```diff
@@ -1,5 +1,21 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
+"""
+The Dates module provides `Date`, `DateTime`, `Time` types, and related funtions.
```

Comment:
Oh, and `Dates` and `Pkg` should be in backquotes since these are Julia objects.